lte/gateway/python/magma/pipelined/qos/qos_tc_impl.py

```diff
@@ -203,6 +211,7 @@ def __init__(self,
         self._uplink = config['nat_iface']
         self._downlink = config['enodeb_iface']
         self._max_rate = config["qos"]["max_rate"]
+        self._enable_pyroute2 = config["qos"].get('enable_pyroute2', False)
```

Comment:
should this be true by default?

Reply:
I still want to make sure all tests works fine before making it default